x-pack/plugins/ml/public/application/components/ml_inference/components/review_and_create_pipeline.tsx

```diff
@@ -0,0 +1,179 @@
+
+export const ReviewPipeline: FC<Props> = ({
+  inferencePipeline,
+  modelType,
+  pipelineName,
+  pipelineCreated,
+  pipelineError,
+}) => {
+  const {
+    services: {
+      docLinks: { links },
+    },
+  } = useMlKibana();
+
+  const inferenceProcessorLink =
+    modelType === 'regression'
+      ? links.ingest.inferenceRegression
+      : links.ingest.inferenceClassification;
+
+  const accordionId = useMemo(() => htmlIdGenerator()(), []);
+
+  const configCodeBlock = useMemo(
+    () => (
+      <EuiCodeBlock language="json" isCopyable overflowHeight="400px">
+        {JSON.stringify(inferencePipeline ?? {}, null, 2)}
+      </EuiCodeBlock>
+    ),
+    [inferencePipeline]
+  );
+
+  return (
+    <>
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem grow={3}>
+          {pipelineCreated === false ? (
+            <EuiTitle size="s">
+              <h4>
+                {i18n.translate(
+                  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.review.title',
+                  {
+                    defaultMessage: "Review the pipeline configuration for '{pipelineName}'",
+                    values: { pipelineName },
+                  }
+                )}
+              </h4>
+            </EuiTitle>
+          ) : null}
+          <>
+            <EuiSpacer size="s" />
+            {pipelineCreated === true && pipelineError === undefined ? (
+              <EuiCallOut
+                title={i18n.translate(
+                  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.create.successMessage',
+                  {
+                    defaultMessage: "'{pipelineName}' has been created successfully.",
+                    values: { pipelineName },
+                  }
+                )}
+                color="success"
+                iconType="check"
+              >
+                <p>
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.create.nextStepsMessage"
+                    defaultMessage="You can now use this pipeline to infer against new ingested data or infer against existing data by {reindexLink} with the pipeline."
+                    values={{
+                      reindexLink: (
+                        <EuiLink href={links.upgradeAssistant.reindexWithPipeline}>
+                          {'reindexing'}
+                        </EuiLink>
+                      ),
+                    }}
+                  />
+                </p>
+              </EuiCallOut>
+            ) : null}
+            {pipelineError !== undefined ? (
+              <EuiCallOut
+                title={i18n.translate(
+                  'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.create.failureMessage',
+                  {
+                    defaultMessage: "Sorry, unable to create '{pipelineName}'.",
```

Comment:
```suggestion
                    defaultMessage: "Unable to create '{pipelineName}'.",
```

Reply:
Updated in 64ba3ac4ec279eb78f5f8a97d01005991ae76219